### ISSUE TITLE
subdomain can be accessed same as subroute

### ISSUE THREAD

**Issue by paulius1230** [hieuhani/nuxt-multi-tenancy#26]
I can access blog.localhost:3000 as well as localhost:3000/blog
but I want to limit it only to blog.localhost:3000 so if user tries to access subroute localhost:3000/blog it would need to display page not found. Is there a way to fix this?

**Comment by hieuhani** [comment]
Yes, I see the issue here. I have a solution to add an option as a restricted mode for these routes that already had the sub-domain route. I will notice you when I'm done

**Comment by DeVoresyah** [comment]
> Yes, I see the issue here. I have a solution to add an option as a restricted mode for these routes that already had the sub-domain route. I will notice you when I'm done

is there any updates on this? do you accept bounty to make it more faster?

**Comment by hieuhani** [comment]
Hi @paulius1230 @DeVoresyah I've released the update nuxt-multi-tenancy@1.2.4 please update.

Thanks,

**Comment by hieuhani** [comment]
@paulius1230 Sorry for the late reply, I'm on a vacation so I will fix them all when I return. I will release the update that breaks nothing, so you can temporarily use the package, and wait for the update later. If you spot out any other edge cases, please let me know. Thank you so much.

**Comment by dhorminick** [comment]
any update on this issue please?
